### PR TITLE
build(aio): display parameter info in a table

### PR DESCRIPTION
**Pull request by petebacondarwin**

### CODE REVIEW

File: packages/animations/src/animation_metadata.ts [hunk 1]
```diff
@@ -354,8 +354,10 @@ export interface AnimationStaggerMetadata extends AnimationMetadata {
  animateChild animateChild}.
  *
  * @experimental Animation support is experimental.
+ * @param name
+ * @param definitions The metadata that defines this animation
  */
-export function trigger(name: string, definitions: AnimationMetadata[]): AnimationTriggerMetadata {
+export function trigger(/** the namexxx*/name: string, definitions: AnimationMetadata[]): AnimationTriggerMetadata {
```
Comment (petebacondarwin) [inline packages/animations/src/animation_metadata.ts]: The changes to this file are purely to test what the result of this new feature looks like. I will revert these once we are happy with the functionality.

**Comment by petebacondarwin** [comment]
Also fixes #21451

**Review by gkalpak — COMMENTED** [review]
I think the descriptions would look better if vertically centered and the height of each row could be reduced.
Other than that LGTM.

params

**Comment by petebacondarwin** [comment]
Regarding the visuals... I do not expect it to look like this. This is more of a PoC.
There is currently a big redesign of the API pages going on, so how this will look will be reliant upon the result of that work. CC @sjtrimble and @jenniferfell

**Comment by petebacondarwin** [comment]
Closing in favour of https://github.com/angular/angular/pull/21874

**Pull request closed without merging**